### ISSUE TITLE
 Out of memory or system resources. Close some windows or programs and try again.  

### ISSUE THREAD

**Issue by BrettMichlitsch** [phw198/OutlookGoogleCalendarSync#1523]
:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.9.5.0
**Installed or Portable**: Installed
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9330002/OGcalsync.log)

**Problem Description**  
Error syncing from google to outlook.

Sync version: 2.9.5.0

Profile: Default

Sync started at 8/12/2022 2:16:14 PM

Syncing from 8/12/2021 to 8/13/2023

Outlook ← Google

Finding Calendar Entries
Scanning Outlook calendar...

4426 Outlook calendar entries found.

Scanning Google calendar...

Authenticating with Google
Preparing to authenticate with Google.
Handshake successful.
8519 Google calendar entries found.

22 are exceptions to recurring events.

Total inc. recurring items spanning sync date range...
Outlook 4426, Google 8519
Checking for orphaned Outlook items...

Failure processing Outlook item:-

Unable to reclaim orphan calendar entries in Outlook calendar.

Out of memory or system resources. Close some windows or programs and try again.

Sync aborted after 1 failed attempts!
Another sync has been scheduled to automatically run in 15 minutes time.

**Steps to Reproduce the Issue**  
1. Start Sync

**Any other information**  
Started happening a few days ago. I checked and am not close to being out of memory or cpu. Regardless I rebooted and tried it with no other programs running and the error still occurs.

**Comment by phw198** [comment]
If you want to sync 8000 items I guess you'll have to put more memory in your machine? See the [FAQ](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#why-cant-i-sync-more-than-1-year-past-or-future) as to whether doing a export/import would be better first and then syncing a smaller date range?

**Comment by BrettMichlitsch** [comment]
It wasn't a memory problem with my computer. I have 32GB of RAM and only half was being used. Perhaps it was a memory leak in the library used. I figured out how to reproduce the issue, it was a specific event causing the problem. Here are the steps: I created an event in google. google->outlook sync. Then I dragged a google event in outlook to a new date and sync google->outlook only. It hangs then that memory error appears during the cleanup part of the sync.
